test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -60,6 +77,12 @@ public void listCommand_parsedCorrectly() {
     }
 
     @Test
+    public void listSortedCommand_parsedCorrectly() {
+        final String input = "listsorted";
+        parseAndAssertCommandType(input, ListSortedCommand.class);
+    }
```

Comment:
As you are adding a totally new command, I would expect to see more JUnit tests than this.